src/index.html

```diff
@@ -6,9 +6,50 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__image">
+        <a class="home__link" href="#home">
+          <img src="images/logo.png" alt="MOYO logo Image">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item is-active">
+            <a class="nav__link" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" data-qa="hover"
+              href="#laptops&computers">Laptops & Computers</a>
```

Comment:
```suggestion
            <a 
              class="nav__link" 
              data-qa="hover"
              href="#laptops&computers"
            >
              Laptops & Computers
            </a>
```

Reply:
fixed one, broke the other..)
all fixed, thank you!